tests/network/l2_bridge/utils.py

```diff
@@ -14,6 +15,7 @@
     NODE_TYPE_WORKER_LABEL,
     SRIOV,
     TIMEOUT_1MIN,
+    TIMEOUT_2MIN,
```

Comment:
I usually nack usages of these useless consts, but in case the existing module already uses it, consistency wins.